sector-builder/src/builder.rs

```diff
@@ -168,6 +168,14 @@ impl SectorBuilder {
     pub fn get_sector_class(&self) -> SectorClass {
         self.sector_class
     }
+
+    pub fn get_all_sealed_sector_status(&self) -> Result<Vec<SectorStatus>> {
+        log_unrecov(self.run_blocking(Request::GetAllSealedSectorStatus))
+    }
+
+    pub fn get_sector_status(&self, sector_id: SectorId) -> Result<SectorStatus> {
```

Comment:
BLOCKING: I'd drop this method. If you want to expose the health of a single sector by its id, write a more general `get_sealed_sector_by_id` method which returns `Result<SealedSectorMetadata>` (and add a `SectorStatus` field to `SealedSectorMetadata`.

Reply:
fine with me